config/typescript/itnl.d.ts

```diff
@@ -1,5 +0,0 @@
-declare namespace Intl {
```

Comment:
this definition limited the options for formatDate options to just `hourCycle` and